### PR TITLE
HTBHF-2406 Add missing integration test for HTBHF-1757 where the claimant…

### PR DESCRIPTION
**Pull request by mikeSmith08**
…was not pregnant with children in previous cycle but none in this cycle and DWP reports they are no longer eligible.

Also did some tidying of constant values in the class for clarity.

### CODE REVIEW

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java [hunk 1]
```diff
@@ -304,13 +306,14 @@ void shouldTestClaimantBecomingIneligible(List<LocalDate> previousCycleChildrenD
...
         return Stream.of(
-                Arguments.of(singletonList(THREE_YEAR_OLD), singletonList(THREE_YEAR_OLD)),
-                Arguments.of(emptyList(), singletonList(THREE_YEAR_OLD)),
-                Arguments.of(singletonList(THREE_YEAR_OLD), emptyList()),
-                Arguments.of(emptyList(), emptyList())
+                Arguments.of(SINGLE_THREE_YEAR_OLD, SINGLE_THREE_YEAR_OLD, EXPECTED_DELIVERY_DATE),
+                Arguments.of(emptyList(), SINGLE_THREE_YEAR_OLD, EXPECTED_DELIVERY_DATE),
+                Arguments.of(SINGLE_THREE_YEAR_OLD, emptyList(), EXPECTED_DELIVERY_DATE),
+                Arguments.of(emptyList(), emptyList(), EXPECTED_DELIVERY_DATE),
+                Arguments.of(SINGLE_THREE_YEAR_OLD, emptyList(), NOT_PREGNANT)
```
Comment (mikeSmith08) [inline api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java]: This is the extra test where the claimant has children in the previous cycle, none in the current cycle and are not pregnant. In all of the original scenarios the claimant was pregnant.

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java [hunk 2]
```diff
@@ -60,7 +60,10 @@
     private static final LocalDate TURNS_FOUR_IN_FIRST_WEEK_OF_NEXT_PAYMENT_CYCLE = START_OF_NEXT_CYCLE.minusYears(4).plusDays(4);
     private static final LocalDate SIX_MONTH_OLD = LocalDate.now().minusMonths(6);
     private static final LocalDate THREE_YEAR_OLD = LocalDate.now().minusYears(3);
+    private static final List<LocalDate> SINGLE_THREE_YEAR_OLD = singletonList(THREE_YEAR_OLD);
     private static final int CARD_BALANCE_IN_PENCE_BEFORE_DEPOSIT = 88;
+    private static final LocalDate EXPECTED_DELIVERY_DATE = LocalDate.now().plusMonths(4);
```
Comment (YetAnotherMatt) [inline api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java]: Is it worth renaming this to something like `DUE_DATE_IN_4_MONTHS`?

Comment (mikeSmith08) [reply api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java]: Yep, sure thing.
